src/app/report/report.component.css

```diff
@@ -149,4 +149,9 @@
   padding-right: 5px;
   /* height: 100%; */
   /* flex: 1 1 auto; */
-}
\ No newline at end of file
+}
```

Comment:
You might wanna not use margins here. You play around with a few CSS properties. See what happens when you use on the parent div,
```css
display: flex;
justify-content: center;
align-items: center;
flex-direction: column;
```

Try it out! You want to try to make it as responsive as possible!